src/rdma.c

```diff
@@ -1689,25 +1697,21 @@ static int rdmaProcessPendingData(void) {
     listNode *ln;
     rdma_connection *rdma_conn;
     connection *conn;
-    listNode *node;
     int processed;
 
     processed = listLength(pending_list);
     listRewind(pending_list, &li);
     while ((ln = listNext(&li))) {
         rdma_conn = listNodeValue(ln);
         conn = &rdma_conn->c;
```

Comment:
How did this work before when  conn was closed and freed? was `rdma_conn->c` NULL ?

Reply:
A conn will be linked into pending list on:
* a write handler is set. because there is no `POLLOUT` event for RDMA. Try to call write handler before event loop.
* RDMA disconnected CM event. This is async event from data plane, use read/write handler to handle this by networking common logic.